src/providers/local/staking/userUserStakingData.ts

```diff
@@ -59,63 +57,15 @@ export type PoolGaugesResponse = {
   pools: PoolGaugesInfo[];
 };
 
-export type UserStakingDataResponse = {
```

Comment:
[Using TS type inference with ReturnType](https://github.com/balancer-labs/frontend-v2/pull/2608/files#diff-431f5029dfdaefe2a611e7169e9fb4b62bbe00314e90efb9c8dfde20b5acffd0) avoids manually maintaining this complex type. The only drawback is that I had to move the related comments close to the related declarations. In any case, most of those comments should not be needed in case of proper naming.